https://github.com/rvoicilas/inotify-tools/pull/97

commit 7ba155e85185dbdcc1f367b847096ef35132d9e3
Author: Bernhard M. Wiedemann <bwiedemann@suse.de>
Date:   Fri Feb 15 19:36:53 2019 +0100

    Use ChangeLog date instead of build date
    
    in order to make builds reproducible.
    See https://reproducible-builds.org/ for why this is good.

    Also use ISO-8601 date format to be readable worldwide.
    
    This date call works with GNU date and FreeBSD date.

diff --git a/configure.ac b/configure.ac
index c2f98f9..cef645a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -65,7 +65,7 @@ AC_C_INLINE
 AC_CHECK_FUNCS([daemon])
 
 # Set variables used in man page templates
-MAN_DATE=$(date +'%B %d, %Y')
+MAN_DATE=$(date -u -r ChangeLog +'%Y-%m-%d')
 MAN_PACKAGE_VERSION=$PACKAGE_VERSION
 AC_SUBST([MAN_DATE])
 AC_SUBST([MAN_PACKAGE_VERSION])
